bin/test

```diff
@@ -6,6 +6,9 @@ go get -t ./...
 
 rm -f coverage.txt
 
+unset AWS_ACCESS_KEY_ID
+unset AWS_SECRET_ACCESS_KEY
```

Comment:
I could be missing some context but seems like the test case should handle that via `os.Unsetenv("VAR")`. Wondering about the side effects of including this in the test script.